lib/ILDAPWrapper.php

```diff
@@ -27,6 +27,12 @@
 namespace OCA\User_LDAP;
 
 interface ILDAPWrapper {
+	const LDAP_SUCCESS = 0x00;
```

Comment:
Better include a link to the documentation where these codes come from. I guess they're error codes from the ldap library, but it's difficult to know, plus I don't know if there are more codes that could be used (not now), or if we could use whatever we want.

Reply:
done